tests/unit/detection/test_detection_notebooks.py

```diff
@@ -78,6 +78,21 @@ def test_02_notebook_run(detection_notebooks, tiny_od_mask_data_path):
     assert len(set([len(d) for d in training_aps])) == 1
 
 
+@pytest.mark.gpu
+@pytest.mark.notebooks
+def test_03_notebook_run(detection_notebooks):
+    notebook_path = detection_notebooks["03"]
+    pm.execute_notebook(
+        notebook_path,
+        OUTPUT_NOTEBOOK,
+        parameters=dict(PM_VERSION=pm.__version__),
```

Comment:
Similar to the test above, can you please here use a dataset with only few images, and only low image resolution, to speed up this test especially on CPU machines. E.e.
   DATA_PATH=tiny_od_mask_data_path,
   IM_SIZE=100,

Reply:
I will add this in next PR, since this PR does not include any tuning part and dataset.  :smile: